### DISCUSSION TITLE
Navigation voice cuts out audio

### DISCUSSION THREAD

**Discussion by Europa284** [osmandapp/OsmAnd#11281] [Q&A]
I have an android head unit with osamd nav in it. When the nav voice interrupts the radio or music audio the music does not return after the voice is done. I have to return to the radio and it comes back on and then switch back to osamd. Next voice nav does the same thing. Is there a way to make the audio return after nav interrupts?

**Comment by barteis** [comment]
You might take a look at Settings/<select a profile>/Navigation settings/Vice prompts.  On mine, Vice guidance output = Media Navigation audio and Pause music is disabled (although you would think it should be enabled). This works fine for me with Pocket Cast and Audible. Both apps stop for the interrupt, then resume from where they stopped. Perhaps this setting or some variation will solve the problem.

**Comment by Europa284** [comment]
I think I did but I'll check Sent from my Galaxy
-------- Original message --------From: Bart Eisenberg ***@***.***> Date: 4/1/21  8:07 PM  (GMT-05:00) To: osmandapp/OsmAnd ***@***.***> Cc: Europa284 ***@***.***>, Author ***@***.***> Subject: Re: [osmandapp/OsmAnd] Navigation voice cuts out audio (#11281) 
You might take a look at Settings//Navigation settings/Vice prompts.  On mine, Vice guidance output = Media Navigation audio and Pause music is disabled (although you would think it should be enabled). This works fine for me with Pocket Cast and Audible. Both apps stop for the interrupt, then resume from where they stopped. Perhaps this setting or some variation will solve the problem.

**Comment by Europa284** [comment]
I've tried everything I can think of. Tried Google maps and hate it but it plays alerts on top of music. Is there a way to do that with osamd?Sent from my Galaxy
-------- Original message --------From: Bart Eisenberg ***@***.***> Date: 4/1/21  8:07 PM  (GMT-05:00) To: osmandapp/OsmAnd ***@***.***> Cc: Europa284 ***@***.***>, Author ***@***.***> Subject: Re: [osmandapp/OsmAnd] Navigation voice cuts out audio (#11281) 
You might take a look at Settings//Navigation settings/Vice prompts.  On mine, Vice guidance output = Media Navigation audio and Pause music is disabled (although you would think it should be enabled). This works fine for me with Pocket Cast and Audible. Both apps stop for the interrupt, then resume from where they stopped. Perhaps this setting or some variation will solve the problem.

**Reply by barteis** [reply]
Other than the settings I mentioned, I don't have any other ideas.

**Comment by Europa284** [comment]
I even turned off the interrupt music on alerts but still shuts it off. Sent from my Galaxy
-------- Original message --------From: Bart Eisenberg ***@***.***> Date: 4/4/21  2:18 PM  (GMT-05:00) To: osmandapp/OsmAnd ***@***.***> Cc: Europa284 ***@***.***>, Author ***@***.***> Subject: Re: [osmandapp/OsmAnd] Navigation voice cuts out audio (#11281) 
Other than the settings I mentioned, I don't have any other ideas.
